flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
Yes. I can look at supporting aarch64 in a future release. I did ask in another comment whether you'd prefer the CEF component to be built from source instead of downloading a binary, knowing that this process takes around ten hours on an average desktop.

Reply:
building from source is always better.

Reply:
Alright if you're sure, I'll set that up in a future version.